### PR TITLE
Added ansible-lint version 6.16.2 test and validation script

### PR DESCRIPTION
**Pull request by ashwinik30**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: a/ansible-lint/build_info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "ansible-lint",
+    "github_url": "https://github.com/ansible/ansible-lint",
+    "version": "main",
```
Comment (nikhil-kalbande) [inline a/ansible-lint/build_info.json:4]: This should be latest version for which build script is tested. Currency jobs expect to run on release branches/tags

File: a/ansible-lint/ansible-lint_ubi_8_7.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+yum install java-17-openjdk-devel openssl-devel git wget tar python39-devel.ppc64le gcc rust cargo  gcc-c++ cmake.ppc64le -y
+pip3 install tox build
+
+#Need to increase npm socket timeout as depdencies require more time
+yum install npm -y
+npm config set fetch-retry-mintimeout 20000
+npm config set fetch-retry-maxtimeout 120000
```
Comment (nikhil-kalbande) [inline a/ansible-lint/ansible-lint_ubi_8_7.sh:47]: do we need both lines to see different values. I think we may just need either of it

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: a/ansible-lint/build_info.json (hunk 1 above)
Comment (ashwinik30) [reply a/ansible-lint/build_info.json:4]: got it , forgot to change here , checking lastest version  in build script

File: a/ansible-lint/ansible-lint_ubi_8_7.sh (hunk 2 above)
Comment (ashwinik30) [reply a/ansible-lint/ansible-lint_ubi_8_7.sh:46]: we need to increase timeout to resolve socket timeout issue , minimum we might not needed .. removed

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**